### DISCUSSION TITLE
Operator always online when browser closed

### DISCUSSION THREAD

**Discussion by ozadorozhnykh** [LiveHelperChat/livehelperchat#2146] [General]
Hello, dear developers and users!

Could you please tell me if it's possible to keep the operator online for 99,999 hours after the browser window is closed, and is this even feasible?

Thank you in advance!

**Comment by remdex** [comment]
Hi,

Yes you ca do that in lhc back office by changing online mode to always online. You can also just set department online hours to 24-7

**Reply by ozadorozhnykh** [reply]
Ou... thank u , i will  try
